### ISSUE TITLE
Build fails while installing 0.3.5 through crate on Pop!OS 21.04

### ISSUE THREAD

**Issue by aquatix** [mdzk-rs/mdzk#13]
```
   Compiling mdzk v0.3.5
error[E0658]: use of unstable library feature 'osstring_ascii'
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/mdzk-0.3.5/src/utils.rs:61:31
   |
61 |                 a.file_name().to_ascii_uppercase().cmp(&b.file_name().to_ascii_uppercase())
   |                               ^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #70516 <https://github.com/rust-lang/rust/issues/70516> for more information

error[E0658]: use of unstable library feature 'osstring_ascii'
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/mdzk-0.3.5/src/utils.rs:61:71
   |
61 |                 a.file_name().to_ascii_uppercase().cmp(&b.file_name().to_ascii_uppercase())
   |                                                                       ^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #70516 <https://github.com/rust-lang/rust/issues/70516> for more information

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
error: could not compile `mdzk`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `mdzk v0.3.5`, intermediate artifacts can be found at `/tmp/cargo-installOA68P1`

Caused by:
  build failed
```

**Comment by kmaasrud** [comment]
I think `osstring_ascii` is stabilized, so you might only have to run a `rustup update` to update your Rust installation, and try installing again. Please follow up if that does not work.

Either way, I think this ASCII conversion can be circumvented, so this will most likely be a non-issue in the next minor release :smile:

**Comment by kmaasrud** [comment]
As of 0.3.6, this should now be fixed. Please reopen if the issue persists

**Comment by aquatix** [comment]
Ah cool, thanks!

I noticed publishing 0.3.6 is failing: https://github.com/mdzk-rs/mdzk/runs/3507327424 so I'll wait while that is resolved.

Thanks for creating this handy tool!

**Comment by kmaasrud** [comment]
@aquatix, yes we've moved to a monorepo and had some issues with publishing. All fixed now, 0.3.7 is published and gtg 👍

**Comment by aquatix** [comment]
Cool, thanks, and good to hear the migration was successful :)
